config/symbols.pspeu.txt

```diff
@@ -70,19 +70,11 @@ g_CdStep = 0x91CE5E0;
 g_unkGraphicsStruct = 0x091CE5F8;
 D_80097488 = 0x091CE688;
 g_randomNext = 0x091CE6B0;
-g_Settings = 0x091CEEBC;
+g_Settings = 0x091CEEBC; // size:0x110
 g_PlayableCharacter = 0x091CF020;
 g_Servant = 0x091CF028;
-g_Player = 0x091CF030;
-g_Player_pl_vram_flag = 0x091CF380;
```

Comment:
One of the downsides to this approach is that you will no longer get direct symbol hints as to the property by mapping them in this manner.

However most structs are not mapped out in much detail this way, and personally I think the benefits of having the whole range of the struct covered rather than just the members we map manually outweighs this.